src/redux/rockets/RockectReducer.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

Reply:
Solved